build_windows.cmd

```diff
@@ -10,13 +10,14 @@ mkdir build
 cd build
 
 set ARCHITECTURE=x64
-rem set BUILD_TYPE=Debug
+rem set BUILD_TYPE=Release
 set BUILD_TYPE=Release
-set VCPKG_PATH=C:\src\vcpkg
+set VCPKG_PATH=C:\DarknetOpenCV\vcpkg

```

Comment:
Unfortunately, all my tutorials, web pages, and youtube videos tell people how to put things in C:\src, not C:\DarknetOpenCV.  I'd rather keep C:\src so all those tutorials are still relevant.